LibSerialize.lua

```diff
@@ -120,6 +120,20 @@ end
 
     Calls `SerializeEx(opts, ...)` with the default options (see below)
 
+* **`LibSerialize:SerializeChunks(...)`**
+
+    Arguments:
+    * `...`: a variable number of serializable values
+
+    Returns:
+    * `coroutine_handler`: function to run the routine. This
```

Comment:
Avoid "coroutine" in your descriptions here since that's just an implementation detail.